ChangeLog

```diff
@@ -1,3 +1,13 @@
+Changes in squid-5.7 (05 Sep 2022):
+
+	- Regression Fix typo in manager ACL
```

Comment:
For readability and consistency with existing similar ChangeLog entries:

```suggestion
	- Regression Fix: Typo in manager ACL
```


or


```suggestion
	- Regression fix: Typo in manager ACL
```

or, if we are going by the "use a custom prefix with the commit title" principle:

```suggestion
	- Regression: Fix typo in manager ACL
```


The existing ChangeLog entries are not terribly consistent either but the more readable variants clearly dominate:

| variant | ChangeLog occurrences |
| --- | ---: |
| `- Regression Fix:` | 32 |
| `- Regression fix:` | 25 |
| `- Regression:` | 9 |
| `- Regression [Ff]ix[^:]` | 2 |